### ISSUE TITLE
branch: a11y-tree

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/area-model-common#144]
For https://github.com/phetsims/area-model-introduction/issues/4, and should be used with Scenery's a11y-tree branch. See https://github.com/phetsims/scenery-phet/issues/365.

**Comment by jonathanolson** [comment]
Merged to master, closing.
